examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2966,6 +2966,15 @@ class GLTFParser {
 
 			}
 
+			texture.userData.mimeType = sourceDef.mimeType;
+
+			if ( ! sourceDef.mimeType ) {
+
+				const ext = sourceDef.uri.split( '.' ).pop().toLowerCase();
+				texture.userData.mimeType = ( ext.includes( 'jp' ) ) ? 'image/jpeg' : ext === 'webp' ? 'image/webp' : 'image/png';
```

Comment:
I agree with @donmccurdy: https://github.com/mrdoob/three.js/issues/23587#issuecomment-1052684982

Better to use jpeg in this case.

Reply:
Okay, I took care of that on export like he asked. I figured this was fine to leave in for now since someone could at least query to see if the input happened to be webp, even though it'll export as png.

Reply:
Yeah I think is OK now – we'll set `userData.mimeType = 'image/webp'` sometimes, but GLTFExporter will export those as PNG instead. JPEG would also be fine with me in that case.

Reply:
Ah I see. That nested ternary operator was hard to read.